### PR TITLE
PD-268 Update Set Up Storage Articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit updates the /GettingStarted/Configure/SetUpStorageScale.md article with create pool and create dataset snippets, cleans up the list of tags. 
It also updates the Storage/Pools/CreatePoolScale.md article with a new snippet, CreatePool.md. 
It updates the /Storage/Datasets/DatasetsSCALE.md article with a new snippet, CreateDatasetSCALE.md. 

This change applies to the Master branch as well as 22.12.2 so this should merge into Master.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by micjohnson777** [comment]
This PR is ready to merge into Master if there are no corrections or issues to fix in this content.

File: content/_includes/CreateDatasetSCALE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Select a dataset (root, parent or child), then click **Add Dataset**.
+
+![AddDatasetNameAndOptions](/images/SCALE/22.12/AddDatasetNameAndOptions.png "Add Dataset Name and Options")
+
+Enter a value in **Name**.
+
+Select either **Sensitive** or **Insensitive** from the **Case Sensitivity** dropdown.
+
+Select either **SMB** for the **Share Type** or leave set to **Generic**, then click **Save**.
```
Comment (clairewrtr) [inline content/_includes/CreateDatasetSCALE.md:15]: Do we want to mention the Share Type options are now Generic, SMB and Apps?

**Review by clairewrtr — APPROVED** [review]
Looks good.

**Review by clairewrtr — APPROVED** [review]
Looks good.

**Pull request merged**